README.md

```diff
@@ -1,35 +1,48 @@
-# ww4ff
-Wake word detection modeling for Firefox.
+# Howl
+Wake word detection modeling for Firefox Voice, supporting open datasets like Google Speech Commands and Mozilla Common Voice.
 
-## Getting Started
+## Quickstart Guide
 
-* Clone the repository: `git clone https://github.com/castorini/ww4ff && cd ww4ff`
+### Installation
 
-* [Install docker](https://docs.docker.com/engine/install/) and [enable GPU support](https://cnvrg.io/how-to-setup-docker-and-nvidia-docker-2-0-on-ubuntu-18-04/)
+A proper Pip package is coming soon. 
 
-* `docker build -t ww4ff .`
+1. `git clone https://github.com/castorini/howl && cd howl`
 
-* `./start.sh` will start up the docker. (if docker container is running it will use the same container) Note that /data and current directory will be mounted
+2. Install [PyTorch](https://pytorch.org) by following your platform-specific instructions.
```

Comment:
what about deep speech? after MFA we no longer require it?

deep speech had to be installed explicitly was quite complicated to install the right thing. if we need it. can you add instruction

Reply:
DeepSpeech is no longer needed for alignment.